index.js

```diff
@@ -140,6 +163,33 @@ module.exports = {
     this.emit('postBuild');
   },
 
+  _getFastBootVersion: function() {
+    var VersionChecker = require('ember-cli-version-checker');
+    var checker = new VersionChecker(this);
+    var fastbootVersionChecker = checker.for('fastboot', 'npm');
+
+    return fastbootVersionChecker.version;
+  },
+
+  _getEmberCliFastBootVersion: function() {
+    var projectPath = process.cwd() + '/node_modules/ember-cli-fastboot';
```

Comment:
this isn't the correct path, as `ember s` can be anywhere within the root of your app.